### PR TITLE
website: Release notes 0.11

### PR DESCRIPTION
**Pull request by slemeur**
### What does this PR do?

Adds release note for 0.11 in documentation

### Screenshot/screencast of this PR

Capture d’écran 2023-01-23 à 12 37 37

### CODE REVIEW

File: website/blog/2023-01-18-release-0.11.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+## Release Details
+
+### Air-Gapped installation [#1104](https://github.com/containers/podman-desktop/pull/1104) [#1120](https://github.com/containers/podman-desktop/pull/1120)
+
+For users who are not able to connect to the internet, a new all-in-one (airgap) binary that includes the Podman Desktop application and the Podman binaries is now available for both Mac and Windows. When initializing a new Podman machine using all-in-one binaries, it'll use the embedded files and not grab them from internet: 
+* on Mac, it embeds the qemu FCOS binaries.
+* on Windows, it embeds the WSL binary.
+
+To differentiate the air-gapped binaries from the regular ones because they're containing 'airgap' in their artifactname and are bigger.
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: Missing space in artifactname and worded a little odd. Maybe something like "Air-gapped binaries are bigger than the regular ones, so the artifact names contain 'airgap' to differentiate them."?

File: website/blog/2023-01-18-release-0.11.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+Please note, that those binaries are available only on releases and not the pre-releases.
+
+There is also an optional way to provide a custom Podman machine image in the create machine form. By providing the path to the image you want, Podman Desktop will create a machine with that image. Leaving the field empty will use the default image (the one included in the binary).
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/210508524-45005536-ac74-4074-92c1-2b3ca51d0073.mp4' />
+
+### Feedback within Podman Desktop [#1078](https://github.com/containers/podman-desktop/pull/1078)
+
+Submitting feedback on Podman Desktop is getting easier as it is possible directly. This will help to get more information about the issues you are facing and will help us to improve the tool.
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: "...possible directly from within the tool"?

File: website/blog/2023-01-18-release-0.11.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+This is a warning displayed for Windows users:
+![windows-docker-compatibility](https://user-images.githubusercontent.com/436777/209305744-9056addd-f122-4f75-9e03-ad27a0109375.png)
+
+This is the warning for Mac users:
+![mac-docker-compatibility](https://user-images.githubusercontent.com/6422176/207964861-a9c1f72c-89d6-4816-beab-397af4125620.png)
+
+
+### Allow to toggle on/off the proxy setting [#983](https://github.com/containers/podman-desktop/pull/983)
+
+In some context, users need the ability to disable and re-enable the proxy configuration very quickly, without havint to entirely reconfigure it. This is now possible from the Podman Desktop settings page, where a toggle to enable/disable the proxy configuration has been added.
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: typo: having not havint

File: website/blog/2023-01-18-release-0.11.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+
+### Configure Podman binary path [#941](https://github.com/containers/podman-desktop/pull/941)
+
+Users may use a custom path to the podman binary. An option within settings (Settings -> Preferences) to allow providing an additional path to Podman binary has been added. This is useful for users who have installed Podman using a package manager and want to use the binary provided by the package manager. Additional information is available in the documentation for Mac users: [http://podman-desktop.io/docs/troubleshooting#unable-to-set-custom-binary-path-for-podman-on-macos](http://podman-desktop.io/docs/troubleshooting#unable-to-set-custom-binary-path-for-podman-on-macos)
+  
+![custom-path](https://user-images.githubusercontent.com/6422176/204832063-5858065a-2fc3-49de-8d23-3b99b7d10dbf.png)
+
+### Provide pre-defined registries [#1201](https://github.com/containers/podman-desktop/pull/1201)
+
+The experience of configuring a registry is getting simplified for the most popular ones. The user will be able to select a registry from a pre-defined list and will only have to provide the username and password. The list of pre-defined registries is the following:
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: "The list of pre-defined registries is the following:" -> "The following registries will be pre-defined:"?

File: website/blog/2023-01-18-release-0.11.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+- IBM Container Registry
+- Google Container Registry
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/6422176/213349477-979ee76f-1468-4663-8e49-c21cd6b276cf.mov' />
+
+### UI/UX Improvements 
+
+#### View Pods logs [#1122](https://github.com/containers/podman-desktop/pull/1122)
+
+The pods details view provides the ability to view the logs of each containers who may be running in a pod. Moreover, as it can also be hard to scan and identify from which container the outputs are coming, we appended the container name at the beginning of each output and differentiate each container with a different color.
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: "containers who" -> "containers that"
"from which container the outputs are coming" -> "which container is producing which output"?

File: website/blog/2023-01-18-release-0.11.md [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211531610-2347d302-4918-46ae-a5a2-c80fac0314f5.mp4' />
+
+For containers that exit immediately or short-lived containers, the feedback is also improved and include report of error now provide a better feedback to the user [#1161](https://github.com/containers/podman-desktop/pull/1161).
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211831905-ebf596d5-efc8-4f55-8cb8-3f31655388b9.mp4' />
+
+
+#### Allows to change the default font size for the editor [#1160](https://github.com/containers/podman-desktop/pull/1160)
+
+The editors used is different screen of Podman Desktop, from the inspect screen, to container's outputs and Kubernetes YAML. The default font size is 10 pixels. It's now possible to adjust the font size to the one the one you prefer. This setting is persisted and will be used for all the editors of Podman Desktop and available from the preferences page (Settings -> Preferences).
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: "The editors used is different screen of Podman Desktop, from the inspect screen, to" -> "An editor is used in several screens of Podman Desktop, from the inspect screen to"

File: website/blog/2023-01-18-release-0.11.md [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+
+#### Keep expanded state of pods when refreshing containers [#1042](https://github.com/containers/podman-desktop/pull/1042)
+
+When switching from different screens of the application or simply refreshing the list of containers, the expanded state of each item in the list is now persisted and will be properly restored.
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/207864147-b68ea9bd-0ca9-42dc-882e-b8a705233749.mp4' />
+
+#### Click on the Pod name redirects to the Pod details page [#1159](https://github.com/containers/podman-desktop/pull/1159)
+
+The list of containers displays pods which may also be running as well. Now clicking on the pod name directly redirects to the Pod details page.
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: "The list of containers displays pods which may also be running as well." -> "The list of containers also displays running pods."

File: website/blog/2023-01-18-release-0.11.md [hunk 8]
```diff
@@ -0,0 +1,256 @@
...
+
+The style of the buttons for actions on item in the list of in details pages have been improved. The background has been removed, but to make the hover state more visible, the "hover" circle is visble and the icon's color is also changing.
+
+On lists:
+![list-actions](https://user-images.githubusercontent.com/6422176/205979121-b49a0ddf-03bb-4a4d-8d12-bc8d0bd52387.png)
+
+On details pages:
+![details-actions](https://user-images.githubusercontent.com/6422176/205979123-ea420b17-e834-4029-82eb-22949889eee9.png)
+
+#### Improved alignments in pages with lists [#1182](https://github.com/containers/podman-desktop/pull/1182)
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md:147]: As much as I love my PR getting called out ;-) , in the future we could probably summarize something this minor to "Improved column alignment" in the Notable list.

File: website/blog/2023-01-18-release-0.11.md [hunk 9]
```diff
@@ -0,0 +1,256 @@
...
+
+Content for helping users to migrate from Docker to Podman has been added:
+- [Using the `podman-mac-helper` tool to migrate from Docker to Podman on macOS](https://github.com/containers/podman-desktop/pull/1088)
+- [better identify podman-mac-helper verification steps](https://github.com/containers/podman-desktop/pull/1129)
+- [Emulating Docker CLI using Podman to migrate from Docker to Podman](https://github.com/containers/podman-desktop/pull/1143)
+- [Migrating from Docker section](https://github.com/containers/podman-desktop/pull/1146)
+- [Verifying that your tools are using Podman](https://github.com/containers/podman-desktop/pull/1152)
+- [Importing saved containers](https://github.com/containers/podman-desktop/pull/1144)
+
+Numbers of other improvements have been made to the documentation and the website:
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: "Numbers" -> "Several"

File: website/blog/2023-01-18-release-0.11.md [hunk 10]
```diff
@@ -0,0 +1,256 @@
...
+- Do not use long calls for external clients - [#1151](https://github.com/containers/podman-desktop/pull/1151)
+- Invalid name/tag for images with registries using a different port - [#1127](https://github.com/containers/podman-desktop/pull/1127)
+- Invalid badge count for images when two images have the same tag - [#1124](https://github.com/containers/podman-desktop/pull/1124)
+- Fixed detection of WSL2 - [#981](https://github.com/containers/podman-desktop/pull/981)
+- Handle invalid kubeconfig files instead of crashing - [#953](https://github.com/containers/podman-desktop/pull/953)
+- Removed redundant actions from details pages - [#1200](https://github.com/containers/podman-desktop/pull/1200)
+- Fixed registry URL not aligned with header - [#1205](https://github.com/containers/podman-desktop/pull/1205)
+- Moved SVGs to reusable components - [#1211](https://github.com/containers/podman-desktop/pull/1211)
+
+** Extensions API have also been improved in this new release: **
```
Comment (deboer-tim) [inline website/blog/2023-01-18-release-0.11.md]: "have" -> "has"

File: website/blog/2023-01-18-release-0.11.md [hunk 11]
```diff
@@ -0,0 +1,256 @@
...
+
+
+![img alt](https://user-images.githubusercontent.com/436777/208451335-fc134ad7-21fb-4011-a632-cc681d5787ee.mp4)
+
+
+
+
+## Community Thank You!
+
+🎉 We’d like to say a big thank you to everyone who helped to make Podman Desktop 0.11 even better:
```
Comment (benoitf) [inline website/blog/2023-01-18-release-0.11.md:223]: here is what I have:
```

## New Contributors
* @emmanuelbernard made their first contribution in https://github.com/containers/podman-desktop/pull/962
* @eidottermihi made their first contribution in https://github.com/containers/podman-desktop/pull/974
* @elervik made their first contribution in https://github.com/containers/podman-desktop/pull/1008
* @evanfpearson made their first contribution in https://github.com/containers/podman-desktop/pull/1145
* @lsm5 made their first contribution in https://github.com/containers/podman-desktop/pull/956
* @jeffmaury made their first contribution in https://github.com/containers/podman-desktop/pull/1194
```

it is missing last 2 ?

File: website/blog/2023-01-18-release-0.11.md (hunk 8 above)
Comment (slemeur) [reply website/blog/2023-01-18-release-0.11.md:147]: :)

**Review by benoitf — APPROVED** [review]
needs to be merged using squash

**Pull request merged**